app/src/main/java/com/github/se/project/model/lesson/Lesson.kt

```diff
@@ -16,4 +16,6 @@ data class Lesson(
     val price: Double = 0.0, // Price for the lesson
     val timeSlot: String = "", // Time slot for the lesson (e.g., "30/10/2024T10:00:00")
     val status: LessonStatus = LessonStatus.PENDING, // Status of the lesson
+    val latitude: Double, // Latitude for lesson location
+    val longitude: Double // Longitude for lesson location
```

Comment:
it's fine for me to do add those 2 parameters (long and lat) but I think we could wrap those 2 into a new "Location" data class that has those 2 attributes :) this is not that important tho.. (I think we had a similar data class in the bootcamp)
ie. data class Location(val latitude: Double, val longitude: Double)

Reply:
btw, we could also give those parameters a default value (0.0) so that we do not have to explicitly modify every file that creates a lesson :) (since you already modified all those files it might be useless..)

Reply:
Yes once we add the feature of the instantaneous lessons we will create a data class Location with the propreties lat,longitude and updateTime. As we will implement it next week I will leave it like that for the moment